ConstantineMaliza/capstone-backend
# Blog

#### What does this PR do?
Blog page
#### Description of Task to be completed?
you can create blog edit blog delete blog by unique id or delete all
#### How should this be manually tested?
- clone project 
- run project in editor
- open postman
- use different request GET, PUT, DELETE, POST
- check use  http://localhost/blog
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
N/A
#### Screenshots (if appropriate)
N/A
#### Questions:
